configure.ac

```diff
@@ -430,6 +430,8 @@ MINGW=`echo $host_os | grep mingw32  2>/dev/null`
 AM_CONDITIONAL(MINGW, test ! -z "$MINGW" )
 if test ! -z "$MINGW"; then
 	WSOCKLIB="-lws2_32"
+	LDFLAGS="-no-undefined $LDFLAGS"
+	AC_DEFINE(HAVE_GETTIMEOFDAY)
```

Comment:
It doesn't hate Windows... It just feels a bit - untested? :) Good thing you're reporting on that so stuff can get fixed. Since I don't even have access to a Windows machine nor have much experience with cmake, maybe someone else has a bit more of a clue than I do.

That being said, **CMakeError.log** and **CMakeOutput.log** in <builddir>/CMakeFiles may contain a hint on what's happening with your missing headers.